lib/pages/ask_doubt/widgets/ask_doubt_card.dart

```diff
@@ -0,0 +1,195 @@
+import 'dart:convert';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:school_erp/theme/text_styles.dart';
+
+// Enum for subjects
+enum Subject {
+  science,
+  math,
+  english,
+  literature,
+  understandingTheSelf,
+}
+
+extension SubjectExtension on Subject {
+  String get displayName {
+    switch (this) {
+      case Subject.science:
+        return "Science";
+      case Subject.math:
+        return "Math";
+      case Subject.english:
+        return "English";
+      case Subject.literature:
+        return "Literature";
+      case Subject.understandingTheSelf:
+        return "Understanding The Self";
+    }
+  }
+}
+
```

Comment:
We dont need these all in front end is just displaying. We wont add hardcoded information of subjects in the code. If new subject is added we cannot code it everytime